zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/RestErrorMapper.java

```diff
@@ -90,8 +90,9 @@ public static ProblemDetail mapErrorToProblem(
     }
     return switch (error) {
       case final NotFoundException nfe:
-        yield createProblemDetail(
-            HttpStatus.NOT_FOUND, nfe.getMessage(), RejectionType.NOT_FOUND.name());
+        yield createProblemDetail(HttpStatus.NOT_FOUND, nfe.getMessage(), nfe.getClass().getName());
```

Comment:
I have changed the Problem.title of `NotFoundException` so that it is consistent with other Exceptions below